packages/react-sdk/src/expression.ts

```diff
@@ -209,3 +209,12 @@ export const generateExpression = ({
     },
   });
 };
+
+export const executeExpression = (expression: undefined | string) => {
```

Comment:
how did we execute epressions before? weird we didn't have that eval so far

Reply:
Yes, it won't. This is cli only utility to extract value from excludePageFromSearch if possible. We never needed this in cli because all expressions were executed in generated runtime.